requirements/base.txt

```diff
@@ -1,7 +1,7 @@
 # Requirements for app run
 
-git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
-django-statici18n==1.8.0
-edx-i18n-tools==0.4.7
-Mako==1.0.7
-bleach==2.1.4
+git+https://github.com/edx/xblock-utils.git@2.0.0#egg=xblock-utils==2.0.0
+django-statici18n==1.9.0
+edx-i18n-tools==0.5.3
+Mako==1.1.3
+bleach==3.1.5
```

Comment:
@taniwha, can we try to use the same versions of the libraries used by the Juniper release?

```suggestion
git+https://github.com/edx/xblock-utils.git@2.1.1#egg=xblock-utils==2.1.1
django-statici18n==1.9.0
edx-i18n-tools==0.5.3
Mako==1.1.3
bleach==3.1.5
```